src/cli.rs

```diff
@@ -106,22 +128,44 @@ const HELPER_CMDS: [&str; 3] = ["list_chips", "list_probes", "version"];
 pub fn handle_arguments() -> anyhow::Result<i32> {
     let opts = Opts::parse();
     let verbose = opts.verbose;
+    let mut log_format = opts.log_format.as_deref();
+    let mut host_log_format = opts.host_log_format.as_deref();
+
+    const DEFAULT_LOG_FORMAT: &str = "{L} {s}\n└─ {m} @ {F}:{l}";
+    const DEFAULT_HOST_LOG_FORMAT: &str = "(HOST) {L} {s}";
```

Comment:
I am just wondering why we have a default log format in both `probe-run` and in `defmt-decoder`. But I think it is okay.

Reply:
There might be consumers of `defmt-decoder` other than `probe-run` (such as `defmt-print`) that expect the format to be the same if no `--log-format` option is provided, so that's why it's duplicated there.

Reply:
That's what I thought. But then (in theory) we wouldn't need the default format in `probe-run` as long as it is the same. Anyways, lets keep it like it is.